openvpn/core/flags_node.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
openvpn/core ? These flags are not related to core of openvpn. We tried so hard to move out all code not directly related to openvpn functionality itself.

Reply:
It should belong to Openvpn Adapter in future, we will mote out later